wf-federated/Snakefile

```diff
@@ -0,0 +1,44 @@
+configfile: "/data/config.yaml"
+
+from snakemake.remote.S3 import RemoteProvider as S3RemoteProvider
+
+# any other remote storage is possible as well e.g. FTP
+# Remote storage has to be suported by Snakemake.
```

Comment:
Maybe simplify into one line, e.g.:
"For other supported storage providers (e.g., FTP), see:"

Reply:
Sure.